activestorage/test/controllers/direct_uploads_controller_test.rb

```diff
@@ -12,6 +12,7 @@ class ActiveStorage::S3DirectUploadsControllerTest < ActionDispatch::Integration
 
     teardown do
       ActiveStorage::Blob.service = @old_service
+      ActiveStorage::Blob.last.destroy
```

Comment:
I just realized we're not using `use_transactional_tests` or including `ActiveRecord::TestFixtures`. It might be a better fix to use transactions in tests.

Reply:
The suite passes with transaction-wrapped tests and seems a little faster so I'm going to do that instead for the first commit.